File: java/src/main/java/challenges/aoc2024/day6/Application.java
package challenges.aoc2024.day6;

import challenges.aoc2024.day6.service.DaySixSolver;
import challenges.aoc2024.day6.service.LabService;
import service.TaskSolver;

public class Application {
    public static void main(String[] args) {
        final TaskSolver solver = new TaskSolver(new DaySixSolver(new LabService()));
        solver.solveTasks();
    }
}

File: scala/src/main/scala/challenges/aoc2025/day5/Application.scala
package challenges.aoc2025.day5

import _root_.service.TaskSolver
import challenges.aoc2025.day5.service.{DayFiveSolver, IngredientService}

object Application {
  def main(args: Array[String]): Unit = {
    val solver: TaskSolver = TaskSolver(DayFiveSolver(IngredientService()))
    solver.solveTasks()
  }
}
